frontend/src/components/VideoQualityLabel/VideoQualityLabel.module.css

```
@@ -0,0 +1,30 @@
+.container {
+    /* Flexbox layout with center alignment */
+    border-radius: var(--radius-base);
+    color: var(--grey-10);
+    font-weight: var(--font-regular);
+    transition: all var(--transition-base);
+    border: 1px solid var(--grey-10);
+    font-family: var(--font-family);
+    font-size: 11px;
+    width: fit-content;
+    padding: 10px 6.5px;
+}
+
+.borderOverlay {
+  /* Absolute positioned border */
+}
+
+.textContent {
+  /* Text styling and positioning */
+
+
+}
+
+.dolbyContainer {
+  /* DolbyVision specific container */
+}
+
+.dolbyImage {
+  /* DolbyVision image styling */
+}
```

- You shouldn't leave empty classes in the file.
- Some values you used we have as variables, they have to be used

Here's my suggestion:

```suggestion
.container {
  display: inline-flex;
  flex-shrink: 0;
  padding: var(--space-1) var(--space-2);
  justify-content: center;
  align-items: center;
  gap: var(--space-3);
  color: var(--grey-10);
  font-weight: var(--font-regular);
  border-radius: var(--radius-base);
  border: 1px solid var(--grey-200);
  font-family: var(--font-family);
  font-size: var(--text-sm);
  width: fit-content;
}

```